api/v1beta1/vcdcluster_types.go

```diff
@@ -76,6 +83,8 @@ type VCDClusterSpec struct {
 	ParentUID string `json:"parentUid,omitempty"`
 	// +optional
 	UseAsManagementCluster bool `json:"useAsManagementCluster,omitempty"`
+	// +optional
```

Comment:
there is a work from Yan I think which tries to keep status in sync with spec. So we need similar fields in status as well. @sahithi is that correct?